### PR TITLE
Add plugin hooks to RTD documentation

### PR DESCRIPTION
**Pull request by gnikonorov**
This PR adds information on the plugin hooks to our read the docs documentation.

### CODE REVIEW

**Comment by gnikonorov** [comment]
Sorry for any email spam. This should be good for review/merge @ssbarnea / @BeyondEvil

**Comment by gnikonorov** [comment]
Bump for a review/merge @BeyondEvil and/or @ssbarnea

File: docs/conf.py [hunk 1]
```diff
@@ -8,9 +8,12 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../src/"))
```
Comment (BeyondEvil) [inline docs/conf.py:14]: ugh... isn't there a better way?

Maybe setting the `PYTHONPATH` env var?

**Review by BeyondEvil — APPROVED** [review]
Approved with one hopeful `ugh` 😏

File: docs/conf.py (hunk 1 above)
Comment (gnikonorov) [reply docs/conf.py:14]: I'm not entirely sure @BeyondEvil. I made that change as per https://github.com/pytest-dev/pytest-html/blob/master/docs/conf.py#L7 and also some results on the internet. It seems like this is the preferred way to go about doing this

Comment (BeyondEvil) [reply docs/conf.py:14]: Fair enough @gnikonorov

Comment (gnikonorov) [reply docs/conf.py:14]: Are you ok with me merging this in then @BeyondEvil?

Comment (BeyondEvil) [reply docs/conf.py:14]: Well sure, on one condition. If you, once it's rebased and updated, approve the flaky windows PR. 😛 @gnikonorov

**Pull request merged**